right/src/keyboard_layout.h

```diff
@@ -33,6 +33,32 @@ typedef enum {
     UHK_KEY_LPRESSLAYER,
 } uhk_key_type_t;
 
+enum {
```

Comment:
I understand that bitfields are more versatile than regular values because one can combine them, but in practice the user gains no practical benefits from combining these actions for a single key, and Agent doesn't support their combination either, so I would rather prefer them to be represented as separate values instead of bitfields. The same values could be used as for the mouse actions of the serialized configuration.